forms/static/forms/admin/dependency_fields.js

```diff
@@ -0,0 +1,27 @@
+(function($){
+    $(document).ready(function() {
+        let newspaperperson = $('#id_newspaperperson_set-0-victim_or_survivor_1:checked').length
+        let radioperson = $('#id_radioperson_set-0-victim_or_survivor_1:checked').length
+        let tvperson = $('#id_televisionperson_set-0-victim_or_survivor_1:checked').length
+        let internentperson = $('#id_internetnewsperson_set-0-victim_or_survivor_1:checked').length
+
+        if((newspaperperson > 0) | (radioperson > 0) | (tvperson > 0) | (internentperson > 0)){
```

Comment:
Hey man! nice job here, just wondering why we're using `bitwise or` instead of `logical or`? Any reason for the preference? @esirK

Reply:
Actually I meant to use the `logical or` lemme update it. Thanks.